### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Darleev**
**Description:**
According to nuget.org documentation it is not possible to pass API Token using nuget.config file for Linux or MacOS, and only the way is to use direct -k and -s options for dotnet nuget push command.

[Documentation link](https://github.com/NuGet/Home/pull/10658)
**Related issue:**
https://github.com/actions/setup-dotnet/issues/190

**Check list:**
- [+] Mark if documentation changes are required.
- [+] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

**Review by vsafonkin — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -104,6 +104,15 @@ steps:
...
+
+# Authticates packages to push to NuGet Org
+- uses: actions/setup-dotnet@v1
+  with:
+    dotnet-version: 3.1.x
+- name: Publish the package to nuget.org
+  run: dotnet nuget push */bin/Release/*.nupkg -k $NUGET_AUTH_TOKEN -s https://api.nuget.org/v3/index.json
+  env:
+    NUGET_AUTH_TOKEN: ${{ secrets.NUGET_TOKEN }}
 ```
```
Comment (AlenaSviridenko) [inline README.md:116]: I think we should add a note that this is the only way to push packages to NuGet registry that works on Ubuntu and macOS images. So customers will know why we recommend it

File: README.md [hunk 2]
```diff
@@ -104,6 +104,15 @@ steps:
     NUGET_AUTH_TOKEN: ${{secrets.AZURE_DEVOPS_PAT}} # Note, create a secret with this name in Settings
 - name: Publish the package to Azure Artifacts
   run: dotnet nuget push <my project>/bin/Release/*.nupkg
+
+# Authticates packages to push to NuGet Org
```
Comment (AlenaSviridenko) [inline README.md:108]: Also, there is probably a typo in all these comments: `Authticates` -> `Authenticates`

**Comment by GGG-KILLER** [comment]
Shouldn't the other nuget upload task examples be updated as well, since this seems to be the behavior of the nuget tool itself?

**Comment by Darleev** [comment]
> Shouldn't the other nuget upload task examples be updated as well, since this seems to be the behavior of the nuget tool itself?

I guess other examples should not be updated, since this part is related to nuget.org feed only.

**Pull request merged**